release/pins/requirements-linux.txt

```diff
@@ -0,0 +1,811 @@
+#
```

Comment:
When during our dev cycle do we check in these files? And when do we update + verify + test them?